<?xml version="1.0"?>
<launch>
    <include file="$(find group3_lab7)/launch/headless_simulator.launch">
        <arg name="map" default="$(find group3_lab7)/maps/levine.yaml"/>
    </include>
    <param name="tracked_path" type="string" value="$(find group3_lab7)/paths/levine_resampled.csv" />
    <node pkg="group3_lab7" name="follow_me" type="follow_me.py" output="screen"/>
    <node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure"/>
    <!--
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find group3_lab7)/launch/follow_me_with_local_planner.rviz" output="screen"/>
    -->
    <remap to="/follow_me/goal" from="/motion_planner/goal" />
    <node pkg="group3_lab7" name="motion_planner" type="motion_planner" output="screen"/>

    <remap to="/motion_planner/path_to_goal" from="/pure_pursuit/tracked_path" />
    <node pkg="group3_lab7" name="pure_pursuit" type="pure_pursuit.py" output="screen"/>
</launch>
